### ISSUE TITLE
Cloudflare pages preset not working in v3.8

### ISSUE THREAD

**Issue by okydk** [nuxt/nuxt#23960]
### Environment

```
- Operating System: Darwin
- Node Version:     v18.16.1
- Nuxt Version:     3.8.0
- CLI Version:      3.9.1
- Nitro Version:    2.7.0
- Package Manager:  npm@9.5.1
- Builder:          -
- User Config:      devtools, modules, appConfig, svgo, nitro, $production
- Runtime Modules:  nuxt-svgo@3.5.6
- Build Modules:    -
```

### Reproduction

```
NITRO_PRESET=cloudflare-pages npx nuxt build 
npx wrangler pages dev dist
```

### Describe the bug

Since `v3.8` the `_routes.json` gets extra exclude-params causing error.

The error is:
```
✘ [ERROR] Invalid _routes.json file found at: /Users/stefan/Developer/radio4-website/dist

  Overlapping rules found. Please make sure that rules ending with a splat (eg. "/api/*") don't
  overlap any other rules (eg. "/api/foo"). This applies to both include and exclude rules
  individually.
```

These are the extra params causing errors, and are not included in `v3.7`:
```
"/_nuxt/builds/*",
"/_nuxt/builds/meta/*",
```

**Comment by danielroe** [comment]
Yes, let's track there.

**Comment by pi0** [comment]
The fix is released on [nitro@2.7.2](https://github.com/unjs/nitro/releases/tag/v2.7.2) you can use `npx nuxi upgrade` to have it.
